pandas/core/arrays/categorical.py

```diff
@@ -1773,9 +1766,10 @@ def view(self):
         Returns
         -------
         view : Categorical
-           Returns `self`!
```

Comment:
can you update the doc-string (or just inherit it)

Reply:
see my comment above, dtype needs to be added (but best to just remove the doc-string completely and inherit it)